build/Makefile

```
@@ -0,0 +1,51 @@
+BUILD_IMAGE_NAME := det-core-rs-albatross-image
+TEMP_CONTAINER_NAME := det-core-rs-albatross-container
+
+WORKDIR := /home/nimiq
+
+TARGET_ARCHITECTURE := linux/amd64
+
+.PHONY: all
+all: build
+
+.PHONY: build-image
+build-image:
+	cd .. && \
+	docker build -t $(BUILD_IMAGE_NAME) -f build/Containerfile .
+
+.PHONY: build
+build: build-image copy-artifacts create-digests
+	docker run --platform $(TARGET_ARCHITECTURE) --rm -v $(CURDIR):/home/nimiq $(BUILD_IMAGE_NAME)
+
+.PHONY: temp-container
+temp-container:
+	echo "Creating container $(TEMP_CONTAINER_NAME) from image $(BUILD_IMAGE_NAME)"
+	docker create --name $(TEMP_CONTAINER_NAME) $(BUILD_IMAGE_NAME) tail -f /dev/null
+
+.PHONY: copy-artifacts
+copy-artifacts: clean-temp-container temp-container 
+	docker cp $(TEMP_CONTAINER_NAME):$(WORKDIR)/target/release ../target
```

We need to make sure the `release` directory exists, otherwise `docker cp` will copy the contents of the source directory into `../target`. This per the command documentation [here](https://docs.docker.com/reference/cli/docker/container/cp/#description).
```suggestion
copy-artifacts: clean-temp-container temp-container
        mkdir -p ../target/release
	docker cp $(TEMP_CONTAINER_NAME):$(WORKDIR)/target/release ../target
```